### ISSUE TITLE
github: ".zenodo.json" dynamic overriding

### ISSUE THREAD

**Issue by slint** [zenodo/zenodo#1737]
Sometimes users might just want to keep the default metadata values added by the integration (e.g. related identifiers, collaborators) but just extend the value in some way. This could be done e.g. by using some special placeholder value (e.g. `<DEFAULT>`). Example:

`.zenodo.json`
```javascript
{
  "related_identifiers": [
    {"identifier": "10.1234/foobar", "relation": "references"},
    "<DEFAULT>" // <---- keep default metadata
  ]
}
```

Final metadata:

```javascript
{
  ...
  "related_identifiers": [
    // added by .zenodo.json
    {"identifier": "10.1234/foobar", "relation": "references"},
    // added by default
    {"identifier": "https://github.com/foo/bar/tree/v1.1.0", "relation": "isSupplementTo"}
  ],
  ...
}
```

**Comment by dimpase** [comment]
yes, this would simplify things a lot. E.g. we need to override the contributors list (we can't list our 540 contributors, it seems that zenodo has a limit of 30, anyway) so we'd replace this with "The Developers" or something like this.
Naturally, we can fill in the rest ourselves, but this means at least version, date needs to be filled into the template in the repo (by hand or otherwise).
